directory_sso_api_client/user.py

```diff
@@ -113,14 +109,30 @@ def get_user_lesson_completed(self, sso_session_id, service, lesson_page):
     def set_user_lesson_completed(self, sso_session_id, service, lesson_page, lesson, module):
         authenticator = AuthenticatorNegotiator(sso_session_id=sso_session_id)
         url = self.endpoints['user_lesson_completed']
-        return self.post(url, {
-            'service': service,
-            'lesson_page': lesson_page,
-            'lesson': lesson,
-            'module': module,
-        }, authenticator=authenticator)
+        return self.post(
+            url,
+            {
+                'service': service,
+                'lesson_page': lesson_page,
+                'lesson': lesson,
+                'module': module,
+            },
+            authenticator=authenticator,
+        )
 
     def delete_user_lesson_completed(self, sso_session_id, service, lesson):
         authenticator = AuthenticatorNegotiator(sso_session_id=sso_session_id)
         url = self.endpoints['user_lesson_completed']
         return self.delete(url, {'service': service, 'lesson': lesson}, authenticator=authenticator)
+
+    def get_user_questionnaire(self, sso_session_id, service):
```

Comment:
Just wondering if questionaire is the best choice for name , oh man I hate naming :-(
conceptually from UI it's a questionaire but  I imagine in the BE it also seems like it becomes part of the user's profile. Hence is questionaire a very soft wording for it.

just my 2p worth i'm not as close to it as you guys so if bob's happy i'm happy too :-)

Reply:
I think the naming is fine, it matches a collection of `Question` objects in SSO and is not linked to a user profile in any way. As @bobmeredith is the 'father' of this endpoint, he might have something to add.

Reply:
Seems OK to me - but welcome any suggestion.
I chose 'questionnaire' as it's a bit more general than 'survey' (which is what the VFM thing is variously described as)
I'm sort of expecting this to be used for other things that comprise a list of questions - without really knowing what those things might be.